src/utils/tokenSymbolText.ts

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro'
+
+// Functions to return {tokenSymbol} and/or 'tokens', translated and (possibly) capitalized
+
+export const singleTokenShort = (tokenSymbol?: string, capitalize?: boolean) =>
```

Comment:
**suggestion** what do you think about combining all of these functions into one, with this signature:

```js
export const tokenSymbolText = ({
  tokenSymbol,
  capitalize,
  plural,
  includeTokenSymbol,
}: {
  tokenSymbol?: string
  capitalize?: boolean
  plural?: boolean
  includeTokenSymbol?: boolean
}) => {
  // todo
}
```

In here, I would use as many `if` statements as needed, instead of ternary statements. Nested ternary's get a bit unwieldy and can be harder to follow. If the function is getting too big, we can create smaller functions to handle specific logic, but still only expose this single function.

Let me know what you think 🙇

Reply:
Yeah it's a great idea. Changed it up, thanks!

Reply:
Doneski!